tests/integration/conftest.py

```diff
@@ -201,6 +212,32 @@ def clean_up(self):
         self.remove_keys_and_certificates()
 
 
+@pytest.fixture()
+def fixture_subman(request):
+    """
+    Fixture.
+    Set up the subscription manager on the system. Wrapper around SubscriptionManager class and its methods.
+    Can be parametrized to set the subscription manager to the stagecdn (needed for SCA Enabled accounts):
+    @pytest.mark.parametrize("fixture_subman", ["set_to_stage=true"], indirect=True)
+    """
+    subman = SubscriptionManager()
+
+    subman.set_up_requirements()
+
+    if hasattr(request, "param"):
+        set_to_stage = request.param
+        print(f">>> Using parametrized option requested in the fixture - {set_to_stage}.")
+        if set_to_stage == "set_to_stage=true":
+            subman.set_up_to_stagecdn()
```

Comment:
The fixture is nice, but feels kind of overengineered.
Here's a suggestion, what about just having `set_cdn_to_stage=False` parameter for the `SubscriptionManager.set_up_requirements()` method and condition that inside?
I understand that we would lose some visibility around the setup in tests using the fixture, but that can be tackled in the docstring of the test function.
With that, the fixture usability would narrow down just to subman installation and setting the baseurl to the stage.
But we don't really need to wrap the subman class for any other scenario than the stage setup.
Its methods are either called directly in tests, if needed, or inside another fixtures (pre_register, satellite)
WDYT @kokesak ?
```suggestion
@pytest.fixture()
def fixture_subman():
    """
    Fixture.
    Set up the subscription manager on the system. Wrapper around SubscriptionManager class and its methods.
    """
    subman = SubscriptionManager()

    subman.set_up_requirements(set_cdn_to_stage=True)

```